src/main/java/io/github/nucleuspowered/nucleus/modules/scheduled/config/ScheduledBroadcast.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.modules.scheduled.config;
+
+import ninja.leaping.configurate.objectmapping.Setting;
+import ninja.leaping.configurate.objectmapping.serialize.ConfigSerializable;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+@ConfigSerializable
+public class ScheduledBroadcast {
+
+    @Setting(value = "name", comment = "loc:config.scheduled.broadcast.name")
+    private String name;
+
+    @Setting(value = "message", comment = "loc:config.scheduled.broadcast.message")
+    private String messageToBroadcast = "This is a broadcast message!";
+
+    @Setting(value = "task-config")
+    private ScheduledTaskConfig taskConfig;
```

Comment:
Is this meant to be `null`, or should this have an initial value?

If it can be null, `getScheduledTaskConfig` should return an `Optional` to make it explicit.